app/services/idv/steps/verify_wait_step_show.rb

```diff
@@ -29,8 +29,9 @@ def process_async_state(current_async_state)
       def async_state_done(current_async_state)
         add_proofing_costs(current_async_state.result)
         form_response = idv_result_to_form_response(
-          idv_result: current_async_state.result,
+          result: current_async_state.result,
           state: flow_session[:pii_from_doc][:state],
+          state_id_jurisdiction: flow_session[:pii_from_doc][:state_id_jurisdiction],
```

Comment:
should we do `.dig` here in case the middle hash is nil?

```suggestion
          state_id_jurisdiction: flow_session.dig(:pii_from_doc, :state_id_jurisdiction),
```

Reply:
I'm not sure, I lean towards no because the previous assumption was that it is present? We shouldn't be able to start this step without it 😬

Reply:
that is a good point, I do think `[][]` is easier to read but happens to be riskier sometimes